sample-kotlin/src/main/java/com/fblitho/lithoktsample/errors/TitleComponentSpec.kt

```diff
@@ -27,10 +28,10 @@ object TitleComponentSpec {
 
   @OnCreateLayout
   fun onCreateLayout(c: ComponentContext, @Prop(resType = STRING) title: String): Component =
-      Text.create(c)
-          .text(title)
-          .textStyle(BOLD)
-          .textSizeDip(24f)
-          .positionDip(YogaEdge.BOTTOM, 16f)
-          .build()
+      text(c) {
+        text(title)
+        textStyle(BOLD)
+        textSizeDip(24f)
+        positionDip(YogaEdge.BOTTOM, 16f)
```

Comment:
Since we have named arguments in Kotlin I wonder if it would make sense to add an extension function that would make this line look like this:
`positionDip(bottom=16f)`